tests/testthat/test-regression100Kcases.R

```diff
@@ -0,0 +1,125 @@
+context("Regression with 100,000 cases")
+
+vehicles <- readRDS("vehicles100kNoNA.rds")
```

Comment:
This data file has 5 predictors. The spec wanted 10 and 20 predictors. Could augment the `data.frame` with 15 more predictors via simuation?